tests/integration/test_replicated_merge_tree_s3_zero_copy/test.py

```diff
@@ -145,9 +146,30 @@ def test_insert_select_replicated(cluster, min_rows_for_wide_part, files_per_par
     ) == (3 * FILES_OVERHEAD) + (files_per_part * 3)
 
 
+def remove_leftovers_from_zk(node_data, node_for_query, replica_name):
```

Comment:
Trying to stabilise the test, it was falling before the changes. Maybe we should move it in separate PR.